js/components/frame.js

```diff
@@ -574,6 +570,15 @@ class Frame extends ImmutableComponent {
         windowActions.setBlockedBy(this.frame, 'noScript', e.details[1])
       }
     })
+    this.webview.addEventListener('did-block-run-insecure-content', (e) => {
+      const hack = siteHacks[urlParse(this.props.location).hostname]
```

Comment:
I think the site hacks need to be translated into default rules for contentSettings.js. This will apply the setting after the page has already loaded and the content is blocked